packages/react-strict-dom/src/native/stylex/CSSLengthUnitValue.js

```diff
@@ -9,7 +9,7 @@
 
 import type { SpreadOptions } from './SpreadOptions';
 
-const LENGTH_REGEX = /^([0-9]*[.]?[0-9]+)(em|px|rem|vh|vmax|vmin|vw)$/;
+const LENGTH_REGEX = /^([-+]?)([0-9]*[.]?[0-9]+)(em|px|rem|vh|vmax|vmin|vw)$/;
```

Comment:
I think this only needs the optional sign in the number match, so can be simplified to

```suggestion
const LENGTH_REGEX = /^(-?[0-9]*[.]?[0-9]+)(em|px|rem|vh|vmax|vmin|vw)$/;
```